src/Services/QueuedJobService.php

```diff
@@ -1533,6 +1579,271 @@ private function addJobHandlersToLogger(LoggerInterface $logger, QueuedJob $job,
         }
     }
 
+    /**
+     * Find any eligible jobs that qualify for an automated job retry and schedule their retry
+     *
+     * @throws ValidationException
+     */
+    protected function retryEligibleJobs(int $queueType): void
+    {
+        // Find any broken jobs that qualify for an automated job retry
+        $jobRetrySentinel = (int) static::config()->get('job_retry_sentinel');
+        $jobRetryLimit = (int) static::config()->get('job_retry_limit');
+        $jobRetryStatus = (array) static::config()->get('job_retry_status');
+
+        $jobStatusMap = [];
```

Comment:
Please either rename this or the above `jobRetryStatusMap` in a way that makes it clear which is which. Since this is a long method, it's likely the assignment will be scrolled off screen when looking at the usage of the variable further down, so clear naming is important here to avoid the wrong variable being used.

An alternative would be to use `unset($jobRetryStatusMap[$retryStatus])` inside the loop below instead of assigning a separate map variable.

Another alternative wold be to extract the below loop into its own `private` method, so that this method only ever has the one variable assignment (`$jobRetryStatusMap = $this->getValidRetryStatusMap()` or similar)